src/components/forms/SimpleOrgSelect/tests/index.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import SimpleOrgSelect from '..';
+import defaultProps from '..';
+
+describe('components/forms/SimpleOrgSelect', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    jest.clearAllMocks();
+  });
+
+  it('renders without crashing', () => {
+    shallow(<SimpleOrgSelect />);
+    mount(<SimpleOrgSelect {...defaultProps} />);
+  });
+  it('renders select options correctly', () => {
+    const wrapper = mount(<SimpleOrgSelect {...defaultProps} />);
+
+    expect(wrapper.children().props()).toMatchSnapshot('SimpleOrgSelect Select Props ownProps');
+    expect(
+      wrapper
+        .children()
+        .children()
+        .children()
+        .props()
+    ).toMatchSnapshot('SimpleOrgSelect Select Props');
+  });
```

Comment:
How does this test show that it "select options correctly"?

Reply:
my bad just wanted to test the props